india_compliance/gst_india/doctype/purchase_reconciliation_tool/purchase_reconciliation_tool.js

```diff
@@ -330,15 +333,15 @@ class PurchaseReconciliationTool {
         return options;
     }
 
-    apply_filters(force) {
-        const has_filters = this.filter_group.filters.length > 0;
+    apply_filters(force, custom_filter = null) {
+        const has_filters = this.filter_group.filters.length > 0 || custom_filter;
```

Comment:
This will completely ignore existing filters for data export. 
It's not a good user experience where the user sees filtered data, but the export is for complete data.